application/src/org/commcare/util/CommCareSessionController.java

```diff
@@ -71,16 +71,16 @@ public void populateMenu(List list, String menu, MultimediaListener listener) {
         entryTable.clear();
         menuTable.clear();
         Enumeration en = session.platform.getInstalledSuites().elements();
+        EvaluationContext ec  = session.getEvaluationContext(getIif());
```

Comment:
I think this can have performance implications (IE: Requesting the ec) Maybe we should make it lazily evaluate?